compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/types/BTypeIdSet.java

```diff
@@ -43,7 +43,12 @@ public BTypeIdSet(Set<BTypeId> primary, Set<BTypeId> secondary) {
 
     public BTypeIdSet(Set<BTypeId> primary) {
         this.primary = primary;
-        this.secondary = emptySet;
+        this.secondary = new HashSet<>();
+    }
+
+    public BTypeIdSet() {
```

Comment:
Shall we chain this constructor to the one with the most arguments (line 44)?

Reply:
fixed